ovs/matchparser_test.go

```diff
@@ -68,6 +69,23 @@ func Test_parseMatch(t *testing.T) {
 				UnsetState(CTStateNew),
 			),
 		},
+		{
+			s:           "ct_state=est|trk",
+			m:           ConnectionTrackingState("est", "trk"),
```

Comment:
Is it more accurate that the state this should be `ConnectionTrackingState("+est", "+trk")`?